android/README.md

```diff
@@ -16,11 +16,11 @@ Keyman for Android uses [Sentry](https://sentry.io) for crash reporting at a ser
  
 ### Compiling From Command Line
 1. Launch a command prompt and cd to the directory **keyman/android**
-2. Run the top level build script `./build.sh -debug` which will:
+2. Run the top level build script `./build.sh build:engine build:app --debug` which will:
     * Compile KMEA (and its KMW dependency)
     * Download default keyboard and dictionary resources as needed
     * Compile KMAPro
-    * Note: to force an update to the latest keyboard and dictionary packages, use the `-download-resources` flag.
+    * Note: to force an update to the latest keyboard and dictionary packages, use the `--download-resources` flag.
```

Comment:
@mcdurdin 

Would this be considered a `configure` or `configure`-like step?  This kind of _is_ an out-of-repo dependency, even if not technically a _code_ dependency.

Reply:
Yes, I think it belongs in `configure`, and `configure` should _always_ download them. While we don't have version specs on these resources, unlike with other dependencies, they are certainly dependencies required for a successful build. They just happen to be binary dependencies rather than source ones.

I am a fan of the `configure` step taking a little longer and being a one-stop shop to get all dependencies in order. I am not so keen on doing it piecemeal, because that kind of step-skipping tends to lead to us tripping over ourselves and forgetting to grab something we need.

Reply:
>  and configure should *always* download them

Can we have it download only if the kmp's don't already exist? I'm thinking of offline development environments (e.g. aeroplane)

Reply:
> `configure` requires Internet access: it's downloading 3rd party modules. We can't get away from it. But `configure` will only run in two circumstances:
> 
>     1. it has never been run before (for any of `build.sh configure|build|test|install`)
> 
>     2. you run `build.sh configure` yourself.

With the current structure, there's one extra aspect I believe to be worth mentioning.

1. ... for _any_ of the in-repo projects.

A successful prior run for building Web (for example) will have a nice, in-place `/node_modules` folder, which the `builder_describe` would likely pick up upon.  So... we should be careful to have it condition `configure` on the uniquely-Android part of the configure.